docs/source/Notify/N001.rst

```diff
@@ -0,0 +1,195 @@
+.. include:: _notify_substitutions_n00x.repl
+
+.. _N001_page:
+
+.. role:: raw-html(raw)
+    :format: html
+
+Notifications Email (SMTP)
+===========================
+
+|N001_shortinfo|
+
+Type: |N001_type|
+
+Name: |N001_name|
+
+Status: |N001_status|
+
+GitHub: |N001_github|_
+
+Maintainer: |N001_maintainer|
+
+Used libraries: |N001_usedlibraries|
+
+Introduction
+------------
+
+*Notifications Email (SMTP)* is used to send an email message using ESPEasy rules.
+
+:raw-html:`<br clear="all">`
```

Comment:
The reStructuredText syntax for a blank line is a pipe symbol on its own on a line: `|`

Reply:
Thanks for the tip. I will update the doc files.